dotcom-rendering/src/web/components/FixedMediumSlowVII.tsx

```diff
@@ -0,0 +1,104 @@
+import type { DCRContainerPalette } from '../../types/front';
+import { LI } from './Card/components/LI';
+import { UL } from './Card/components/UL';
+import { FrontCard } from './FrontCard';
+
+type Props = {
+	trails: TrailType[];
+	containerPalette?: DCRContainerPalette;
+	showAge?: boolean;
+};
+
+export const FixedMediumSlowVII = ({
+	trails,
+	containerPalette,
+	showAge,
+}: Props) => {
+	const primary = trails[0];
+	const firstSlice = trails.slice(1, 3);
+	const secondSlice = trails.slice(3, 7);
```

Comment:
Maybe we could keep things consistent?

```suggestion
	const primary = trails.slice(0,1);
	const firstSlice = trails.slice(1, 3);
	const secondSlice = trails.slice(3, 7);
```

Reply:
Thanks for the suggestion! I kept this as `trails[0]` because I wanted primary to be a `TrailType` so that I can call directly `primary.format` etc. `trails.slice(0, 1)` gives me a `TrailType[]` with one element which I will then have to extract from the array or call a `map` on it below to access its properties. For this reason I would like to keep it as it is but let me know if I'm missing something and there is a simpler way of doing this!